Package.swift

```diff
@@ -177,11 +177,13 @@ let package = Package(
         // MARK: SwiftPM specific support libraries
 
         .systemLibrary(name: "SPMSQLite3", pkgConfig: "sqlite3"),
+        .systemLibrary(name: "SPMLibzip", pkgConfig: "libzip"),
```

Comment:
Naming nit: looks like SwiftPM is already an established abbreviation in our codebase and documentation, maybe this could be named `SwiftPMLibzip` for consistency?

Reply:
There's a lot of "SPM" as well, especially in type names, but I agree that we should clean that up at some point.